Formula/libbitcoin-explorer.rb

```diff
@@ -0,0 +1,80 @@
+class LibbitcoinExplorer < Formula
+  desc "Bitcoin Command-line Tool"
+  homepage "https://github.com/libbitcoin/libbitcoin-explorer"
+  url "https://github.com/libbitcoin/libbitcoin-explorer/archive/v3.3.0.tar.gz"
+  sha256 "029dc350497bdaad4d8559f7954405011b9e1b996aa4d4cc124f650e2eca00a6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "zeromq"
+  depends_on "libbitcoin"
+
+  resource "secp256k1" do
+    url "https://github.com/libbitcoin/secp256k1/archive/v0.1.0.13.tar.gz"
+    sha256 "9e48dbc88d0fb5646d40ea12df9375c577f0e77525e49833fb744d3c2a69e727"
+  end
+
+  resource "libbitcoin-protocol" do
+    url "https://github.com/libbitcoin/libbitcoin-protocol/archive/v3.3.0.tar.gz"
+    sha256 "7902de78b4c646daf2012e04bb7967784f67a6372a8a8d3c77417dabcc4b617d"
+  end
+
+  resource "libbitcoin-network" do
+    url "https://github.com/libbitcoin/libbitcoin-network/archive/v3.3.0.tar.gz"
+    sha256 "cab9142d2b94019c824365c0b39d7e31dbc9aaeb98c6b4bf22ce32b829395c19"
+  end
+
+  resource "libbitcoin-client" do
+    url "https://github.com/libbitcoin/libbitcoin-client/archive/v3.3.0.tar.gz"
+    sha256 "ac22793201a269789a5d10e92333a2f59e887256c87c2e72b20ccd023d618757"
+  end
+
+  def install
+    resource("secp256k1").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{libexec}",
+                            "--enable-module-recovery"
+      system "make", "install"
+    end
+
+    ENV.prepend_path "PKG_CONFIG_PATH", "#{libexec}/lib/pkgconfig"
+
+    resource("libbitcoin-protocol").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{libexec}"
+      system "make", "install"
+    end
+
+    resource("libbitcoin-network").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{libexec}"
+      system "make", "install"
+    end
+
+    resource("libbitcoin-client").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{libexec}"
+      system "make", "install"
+    end
+
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bx", "seed"
```

Comment:
Can we get this to verify something successful happened other than just a zero exit code? Perhaps using an assert?